### ISSUE TITLE
Send html tags together with  content

### ISSUE THREAD

**Issue by loocd** [danieldotnl/ha-multiscrape#406]
**Is your feature request related to a problem? Please describe.**
I am trying to scrape a website and retain basic styling (headers, strong, etc.) to put into a markdown card. But multiscrape pulls only the content and leaves me with an unstyled 'mess'. This is an issue especially with larger texts.
also see [Post by myself](https://community.home-assistant.io/t/scrape-sensor-improved-scraping-multiple-values/218350/555) and Post by [Roemer](https://community.home-assistant.io/t/scrape-sensor-improved-scraping-multiple-values/218350/34).

**Describe the solution you'd like**
I would like to have the option to choose whether or not I want html tags included in my scraped content. Then I could add the content to a markdown card and automatically have proper basic styling available.

**Describe alternatives you've considered**
For testing purposes I have manually translated html into commonmark and this works like a charm. But of course this is only for testing and not feasible for scraping live content. But commonmark content in a .txt file retains all styling and allows me to get the expected styling in a markdown card.

**Additional context**
Please feel free to ask for any additional information - happy to provide whatever you need :)

**Comment by loocd** [comment]
I've also tried to set up a basic .txt file containing html - this is not rendered as html but as normal text: 
![image](https://github.com/user-attachments/assets/0e7cabad-988b-4f00-b529-b4ce4c3a603e)


but multiscrape still removes the plain text html tags
![image](https://github.com/user-attachments/assets/1f5c1e0f-687f-4afa-bc4e-9650a0d2c2f1)

**Comment by loocd** [comment]
Did some further investigation:

If I write the *.txt file like this:
![image](https://github.com/user-attachments/assets/c4f3a1fb-cc27-4288-8def-53d074db3356)
Then it get's scraped 'correctly':
![image](https://github.com/user-attachments/assets/3dc47b23-d3e8-4f75-9be5-d16009e11489)


so I tried to write an according value template:
```
- name: My Scraper
  resource: https://web.com/mysite
  scan_interval: 3600
  sensor:
    - unique_id: my_scraper
      name: My Scraper
      select: "section.content__section"
      value_template: "{{ value[:100] }}"
      attributes:
        - name: full_text
          select: "section.content__section"
          value_template: "{{value | replace('<','&lt;') | replace('>','&gt;')}}"
```

Unfortuantely it seems that by the time it gets to the value template, there are already no html tags left.

hope this helps - open for any suggestions!

**Comment by danieldotnl** [comment]
Released in [v7.2.0](https://github.com/danieldotnl/ha-multiscrape/releases/tag/v7.2.0)!

**Comment by loocd** [comment]
this is absolutely amazing, thank you!
